Source/WebCore/Modules/identity/DigitalCredential.h

```diff
@@ -64,6 +69,9 @@ class DigitalCredential final : public BasicCredential {
 private:
     DigitalCredential(JSC::Strong<JSC::JSObject>&&, IdentityCredentialProtocol);
 
+    static ExceptionOr<Vector<ValidatedDigitalCredentialRequest>> validateRequests(const Document&, Vector<UnvalidatedDigitalCredentialRequest>&&);
+    static ExceptionOr<Vector<UnvalidatedDigitalCredentialRequest>> convertObjectsToDigitalPresentationRequests(const Document&, const Vector<DigitalCredentialRequest>&);
+    static ExceptionOr<DigitalCredentialsRequestData> prepareDigitalCredentialRequestData(const Document&, Vector<ValidatedDigitalCredentialRequest>&&);
```

Comment:
I think you can remove this declaration, since it seems to be undefined, and you use the equivalent provided by `CredentialRequestCoordinator`.